### PR TITLE
more docstring goodness

### PR DESCRIPTION
**Pull request by rhsimplex**
more docs more docs

### CODE REVIEW

File: image_match/elasticsearch_driver.py [hunk 1]
```diff
@@ -5,9 +5,44 @@
...
+            doc_type (Optional[string]): a name for the document time (default 'image')
+
+            timeout (Optional[int]): how long to wait on an Elasticsearch query, in seconds (default 10)
+
+            size (Optional[int]): maximum number of Elasticsearch results (default 100)
+
+            *args (Optional): Variable length argument list to pass to base constructor
+
+            **kwargs (Optional): Arbitrary keyword arguments to pass to base constructor
+
```
Comment (vrde) [inline image_match/elasticsearch_driver.py:24]: Can you remove the new lines between the items?

**Comment by vrde** [comment]
some URLs are super long, can you split the string into multiple tokens? If the string is inside parenthesis you can split it like this.

``` python
ses.add_image('https://upload.wikimedia.org/wikipedia/commons/thumb/e/ec/'                                                                                                                                                                                                                                                                                                                                                                
              'Mona_Lisa,_by_Leonardo_da_Vinci,_from_C2RMF_retouched.jpg/'                                                                                                                                                                                                                                                                                                                                                                
              '687px-Mona_Lisa,_by_Leonardo_da_Vinci,'                                                                                                                                                                                                                                                                                                                                                                                    
              '_from_C2RMF_retouched.jpg')                                                                                                                                                                                                                                                                                                                                                                                                
```

**Comment by rhsimplex** [comment]
> some URLs are super long, can you split the string into multiple tokens? If the string is inside parenthesis you can split it like this.

I don't want to do this, because it makes it harder to copy/paste the examples into an interactive interpreter.

**Comment by vrde** [comment]
> I don't want to do this, because it makes it harder to copy/paste the examples into an interactive interpreter.

I understand and I'm fine with that. For future development we should consider keeping docstring within 72 chars.

In the following screenshot you can see the different rendering for two docstrings using the `help` function, `add_image` is 72 chars, `search image` 79.

image

**Pull request merged**